doc/api/process.markdown

```diff
@@ -179,6 +179,55 @@ this, you can either attach a dummy `.catch(() => { })` handler to
 `resource.loaded`, preventing the `'unhandledRejection'` event from being
 emitted, or you can use the [`'rejectionHandled'`][] event.
 
+## Event: 'warn'
+
+Emitted whenever Node.js emits a "Process Warning". Process Warnings are
+similar to Errors in that they describe exceptional conditions that are being
+brought to the user's attention, they are not, however, part of the normal
+Node.js and JavaScript Error handling flow. Node.js currently emits Warnings
```

Comment:
I know it's an American thing but randomly capitalizing words mid-sentence always Strikes me as Somewhat Odd.